mkosi/mkosi.images/build/mkosi.conf.d/debian-ubuntu/mkosi.build.chroot

```diff
@@ -34,6 +34,14 @@ fi
 # We transplant the debian/ folder from the deb package sources into the upstream sources.
 mount --mkdir --bind "$SRCDIR/pkg/$PKG_SUBDIR/debian" "$SRCDIR"/debian
 
+# TODO: drop this workaround when Debian/Ubuntu drops SysV compat
```

Comment:
please drop this, I'll mark these so that they don't get picked by upstream builds

Reply:
Without this, mkosi CI will fail.
If the file lists in debian/ubuntu repository are updated, we can remove the change.
This is a chicken-egg relation. Please revert the change after this is merged and the downstream file lists are updated.

Reply:
Waiting for salsa CI. It's not chicken-and-egg as uninstalled files are just ignored. Should be finished in a bit, I'll reply here once it is

Reply:
Note, such process is not specific to this PR, but usual procedure when we drop/rename components.

Reply:
This is now pushed, so you can run the package update script in this branch and it should all work out, hopefully

Reply:
Also see the semaphore CI and ubuntu failures - this workaround is not sufficient, it only applies to mkosi builds

Reply:
Great. Thank you!

Reply:
You need to run `./tools/fetch-distro.py -u debian` and commit the changes otherwise mkosi won't pick them up

Reply:
> You need to run `./tools/fetch-distro.py -u debian` and commit the changes otherwise mkosi won't pick them up

Ouch. Thanks. Will do.